src/main/scala/it/pagopa/interop/backendforfrontend/server/impl/dependencies/Dependencies.scala

```diff
@@ -83,6 +76,20 @@ trait Dependencies {
   implicit val userRegistryApiKeyValue: UserRegistryApiKeyValue =
     userregistry.client.invoker.ApiKeyValue(ApplicationConfiguration.userRegistryApiKey)
 
+  val rateLimiter: RateLimiter =
+    RedisRateLimiter(ApplicationConfiguration.rateLimiterConfigs, OffsetDateTimeSupplierImpl)
+
+  val rateLimiterDirective: ExecutionContext => Seq[(String, String)] => Directive1[Seq[(String, String)]] = {
+    ec => contexts =>
+      {
+        val logger: LoggerTakingImplicit[ContextFieldsToLog] = Logger.takingImplicit[ContextFieldsToLog](this.getClass)
```

Comment:
not sure if this has downsides 🤔

Reply:
What makes you doubt?

Reply:
Just the fact that we defined a logger in each class, and maybe it had some underlying reason that I'm missing

Reply:
at least we don't instantiate a logger at each call

Reply:
that was the purpose but I forgot to move the log creation 😂 
thanks for pointing it out, I'll fix it!